### PR TITLE
Adding "Add Environmental Variables to Amplify Console Build" to CLI - Import section

### PR DESCRIPTION
**Pull request by mjkubba**
_Issue 1271, if available:_
https://github.com/aws-amplify/amplify-hosting/issues/1271
_Description of changes:_
Adding "Add Environmental Variables to Amplify Console Build" to CLI - Import section

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by offlineprogrammer — APPROVED** [review]

File: src/pages/cli/auth/import.mdx [hunk 1]
```diff
@@ -59,3 +59,14 @@ If you want to have Amplify manage your auth resources in a new environment, run
...
 Run `amplify push` to complete the unlink procedure.
+
+## Add Environmental Variables to Amplify Console Build
+
+In order to successfully build your application with Amplify Console add the following environmental variables to your build environment:
+
+|Environment Variable|Description|
+|-|-|
+|AMPLIFY_USERPOOL_ID|The ID for the Amazon Cognito user pool imported for auth|
+|AMPLIFY_WEBCLIENT_ID|The ID for the app client to be used by web applications.   The app client must be configured with access to the Amazon Cognito user pool specified by the AMPLIFY_USERPOOL_ID environment variable.|
```
Comment (sachscode) [inline src/pages/cli/auth/import.mdx:70]:
```suggestion
|AMPLIFY_WEBCLIENT_ID|The ID for the app client to be used by web applications. The app client must be configured with access to the Amazon Cognito user pool specified by the AMPLIFY_USERPOOL_ID environment variable.|
```

File: src/pages/cli/auth/import.mdx [hunk 2]
```diff
@@ -59,3 +59,14 @@ If you want to have Amplify manage your auth resources in a new environment, run
...
+|AMPLIFY_NATIVECLIENT_ID|The ID for the app client to be used by native applications.   The app client must be configured with access to the Amazon Cognito user pool specified by the AMPLIFY_USERPOOL_ID environment variable.|
```
Comment (sachscode) [inline src/pages/cli/auth/import.mdx:71]:
```suggestion
|AMPLIFY_NATIVECLIENT_ID|The ID for the app client to be used by native applications. The app client must be configured with access to the Amazon Cognito user pool specified by the AMPLIFY_USERPOOL_ID environment variable.|
```

**Review by sachscode — COMMENTED** [review]
Mostly LGTM. Just some spacing issues

**Review by offlineprogrammer — APPROVED** [review]

**Review by sachscode — APPROVED** [review]

**Comment by mjkubba** [comment]
Added cSpell ignore statement at the top of the document to ignore some if the flagged words.

**Pull request merged**